packages/framework/tests/Unit/Pages/BaseHydePageUnitTestMethods.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Hyde\Framework\Testing\Unit\Pages;
+
+interface BaseHydePageUnitTestMethods
+{
+    public function testPath();
+
+    public function testGetBladeView();
+
+    public function testSourcePath();
+
+    public function testFiles();
+
+    public function testNavigationMenuLabel();
+
+    public function testGetOutputPath();
+
+    public function testGet();
```

Comment:
Update test for https://github.com/hydephp/develop/pull/713 and add test for https://github.com/hydephp/develop/pull/712